src/main/java/core/util/mapper/impl/CinemaHallMapperImpl.java

```diff
@@ -0,0 +1,25 @@
+package core.util.mapper.impl;
+
+import core.model.CinemaHall;
+import core.model.dto.CinemaHallRequestDto;
+import core.model.dto.CinemaHallResponseDto;
+import core.util.mapper.CinemaHallMapper;
+import org.springframework.stereotype.Component;
+
+@Component
+public class CinemaHallMapperImpl implements CinemaHallMapper {
+    @Override
+    public CinemaHallResponseDto mapToDto(CinemaHall cinemaHall) {
+        CinemaHallResponseDto chrDto = new CinemaHallResponseDto();
```

Comment:
```suggestion
        CinemaHallResponseDto responseDto = new CinemaHallResponseDto();
```
let's make the variable names readable